readme.md

```diff
@@ -1,4 +1,25 @@
 Binary builds of [`sqlite3`](sqlite.org/) CLI for Linux x86_64.
 
 Versions:
-- [3.33.0](https://github.com/joncfoo/sqlite3-bin/releases/download/3330000/sqlite3)
+- [3.45.1](https://github.com/rstudio/sqlite3-bin/releases/download/3450100/sqlite3)
+- [3.33.0](https://github.com/rstudio/sqlite3-bin/releases/download/3330000/sqlite3)
```

Comment:
Do you know if the precompiled binaries are good enough for our purposes?
https://www.sqlite.org/download.html

Reply:
I'll check and see if it works w/ our centos image

Reply:
Yeah we'll use the pre-compiled Linux binaries from https://www.sqlite.org/download.html
No need to build our own anymore.